### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by notwaldorf**
- I changed the usage in the demo of `iron-meta`, and used the validator object instead. @cdata do you think this is better or worse?

### CODE REVIEW

File: iron-validator-behavior.html [hunk 1]
```diff
@@ -17,36 +17,13 @@
...
    * @polymerBehavior
    */
   Polymer.IronValidatorBehavior = {
-
-    properties: {
-
-      /**
-       * Namespace for this validator.
-       */
-      validatorType: {
```
Comment (bicknellr) [inline iron-validator-behavior.html:31]: Why were these removed?

Comment (notwaldorf) [reply iron-validator-behavior.html:31]: They were unused. `validatorType` was always set to `validator`, and changes to it were not observed, so it was useless as a property (also, it was only used in `ready`, so if you ever did change it, it was too late)

File: bower.json [hunk 2]
```diff
@@ -19,14 +19,32 @@
...
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "iron-meta": "PolymerElements/iron-meta#^1.0.0",
+        "polymer": "Polymer/polymer#^1.1.0"
+      },
+      "devDependencies": {
+        "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
+        "paper-styles": "PolymerElements/paper-styles#^1.0.0",
```
Comment (bicknellr) [inline bower.json]: This used to be `^1.0.4`.

File: iron-validator-behavior.html (hunk 1 above)
Comment (bicknellr) [reply iron-validator-behavior.html:31]: Wasn't it possible to do something like this?
```javascript
MyValidatableBehavior = [Polymer.IronValidatableBehavior, {
  properties: {
    validatorType: {
      type: String,
      value: 'my-validator',
    }
  },
}];
```

File: bower.json [hunk 3]
```diff
@@ -19,14 +19,32 @@
...
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
+  },
+  "resolutions": {
+    "test-fixture": "^2.0.0"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "iron-meta": "PolymerElements/iron-meta#^1.0.0",
+        "polymer": "Polymer/polymer#^1.1.0"
```
Comment (bicknellr) [inline bower.json]: (reminder about updating to `^1.7.0`)

**Comment by bicknellr** [comment]
BTW, the demo says 'The cats-only validator only accepts the string "cats"' but accepts any prefix of 'cats'.

File: iron-validator-behavior.html (hunk 1 above)
Comment (bicknellr) [reply iron-validator-behavior.html:31]: Oh.... https://github.com/PolymerElements/iron-validatable-behavior/pull/29/files#diff-ce0af24a1bb20b227d3d31fea9235f66R67

**Comment by notwaldorf** [comment]
@bicknellr yeah, the demo behaves the same way in 1.0 though. Should I change that string or just leave it?

File: iron-validator-behavior.html (hunk 1 above)
Comment (notwaldorf) [reply iron-validator-behavior.html:31]: Yah. So is this still an issue, or is it ok?

**Comment by notwaldorf** [comment]
@bicknellr done both!

File: iron-validator-behavior.html (hunk 1 above)
Comment (bicknellr) [reply iron-validator-behavior.html:31]: Nah, I'm not worried about removing this if it doesn't do anything.

Comment (bicknellr) [reply iron-validator-behavior.html:31]: Well, is there any reason not to keep them as they are?

**Review by bicknellr — APPROVED** [review]

**Comment by bicknellr** [comment]
Still worth considering leaving the removed properties in IMHO.
